CorsixTH/Lua/hospital.lua

```diff
@@ -183,18 +183,24 @@ function Hospital:Hospital(world, avail_rooms, name)
   self.policies["stop_procedure"] = 1 -- Note that this is between 1 and 2 ( = 100% - 200%)
   self.policies["goto_staffroom"] = 0.6
   self.policies["grant_wage_increase"] = TheApp.config.grant_wage_increase
+
   -- Randomly select three insurance companies to use, only different by name right now.
-  -- The first ones are more likely to come
+  -- The companies in the first quarter of the list are more likely to be selected
```

Comment:
I feel like these two comments go against each other:
1. Insurance companies are random
2. Insurance companies are shifted towards the start of the list.
Unless this was original behaviour we should instead try and make the selection more random. Though it could be out of scope of this PR.